### PR TITLE
Rephrasing around UI and having SCC details before installation.

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

- Removed instruction for obtaining UI,
- Removed obsolete download link.
- ~~Extended note with recommendation to always use UI, except when SUMA image is not available (cloud only).~~

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes partially #14451

### CODE REVIEW

File: modules/installation/pages/general-requirements.adoc [hunk 1]
```diff
@@ -56,7 +57,7 @@ You do not require a separate code for SLES{nbsp}{sles-version} {sp-version}.
 endif::[]
 
 If not already done, download the {sle} Unified Installer from https://download.suse.com/index.jsp.
```
Comment (juliogonzalez) [inline modules/installation/pages/general-requirements.adoc:59]:
```suggestion
If not already done, download the {sle} Unified Installer from https://download.suse.com.
```
Works fine, and we don't care if the index is jsp, php, htmp, py or whatever else :-D

In the end that's working just as a redirect.

File: modules/installation/pages/install-proxy-unified.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {susemgr} Proxy is a {suse} product within the {sle} product family.
 This section describes how to install {susemgr} Proxy from {sle} installation media.
-It assumes you have already registered the {susemgr} Proxy product with the {scc} and have a registration code.
+It assumes you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr} Proxy.
```
Comment (juliogonzalez) [inline modules/installation/pages/install-proxy-unified.adoc:6]: Why do you need the organization credentials to register the proxy?

You just need the registration code.

File: modules/installation/pages/install-server-unified.adoc [hunk 3]
```diff
@@ -8,6 +8,12 @@ For this topic we assume that you already have valid organization credentials wi
 
 For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].
 
+[NOTE]
+====
+The recommended way of installing {susemgr} is from the {sle} installation media and the Unified Installer.
+In case of installing {susemgr} in a public cloud where {susemgr} image is not available, it is possible to change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}.
+For steps how to do that follow the instructions at installation:install-vm.adoc[].
```
Comment (juliogonzalez) [inline modules/installation/pages/install-server-unified.adoc:15]: I am not sure at all about this note.

Reality is:
- Use always the Unified installer unless:
  - You are installing on a cloud where SUSE Manager images are available
  - You are want to install on an VM using JeOS

If the public cloud does not have a SUSE Manager image, I am not sure if what you described will work, or if it's supported at all even if it works.

Keep in mind the link you provide mentions JeOS. I am not sure if the SLE images on public clouds are based on JeOS.

I guess we need to get confirmation from @paususe for this.

Comment (Loquacity) [reply modules/installation/pages/install-server-unified.adoc:15]: I would link to the Public Cloud Quick Start, or the pubcloud pages in the Install Guide. Also, remember the "for more ... see" formulation for xrefs.

File: modules/installation/pages/install-proxy-unified.adoc (hunk 2 above)
Comment (0rnela) [inline modules/installation/pages/install-proxy-unified.adoc:6]:
```suggestion
It assumes you already have obtained a registration code for your {susemgr} Proxy.
```

File: modules/installation/pages/general-requirements.adoc [hunk 4]
```diff
@@ -42,21 +42,24 @@ Create an account with {scc} before installation of {sles} and {productname}.
 
 Depending on your organization's setup, you might also need to activate your subscription, using the btn:[Activate Subscriptions] menu.
 
+For more information about using SCC, go to https://scc.suse.com/docs/help.
```
Comment (keichwa) [inline modules/installation/pages/general-requirements.adoc:45]:
```suggestion
For more information about using SCC, see https://scc.suse.com/docs/help.
```
normalization / simple English

File: modules/installation/pages/general-requirements.adoc [hunk 5]
```diff
@@ -42,21 +42,24 @@ Create an account with {scc} before installation of {sles} and {productname}.
...
 
 //REMARK What about Uyuni?
 ifeval::[{suma-content} == true]
 You only require a valid registration code for {productname}.
 You do not require a separate code for SLES{nbsp}{sles-version} {sp-version}.
 endif::[]
 
-If not already done, download the {sle} Unified Installer from https://download.suse.com/index.jsp.
-Direct link to {sle} {sles-version} {sp-version}, required to install SUSE Manager 4.1: https://download.suse.com/index.jsp?product_id=&search=Search&families=22609&version=68287.
+If not already done, download the {sle} Unified Installer from https://download.suse.com.
```
Comment (keichwa) [inline modules/installation/pages/general-requirements.adoc:59]: Did we already explain somewhere what the Unified Installer is?  Two sentences are probably sufficient: With the Unified Installer you can install all products that are part of SLE. The Unified Installer is SLE installation media.

File: modules/installation/pages/install-server-unified.adoc [hunk 6]
```diff
@@ -8,6 +8,12 @@ For this topic we assume that you already have valid organization credentials wi
 
 For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].
 
+[NOTE]
+====
+The recommended way of installing {susemgr} is from the {sle} installation media and the Unified Installer.
```
Comment (keichwa) [inline modules/installation/pages/install-server-unified.adoc:13]:
```suggestion
The recommended way of installing {susemgr} is from the {sle} installation media with the Unified Installer.
```

File: modules/installation/pages/install-server-unified.adoc (hunk 3 above)
Comment (keichwa) [inline modules/installation/pages/install-server-unified.adoc:15]:
```suggestion
For steps how to do that, see xref:installation:install-vm.adoc[].
```
normalization ;)

File: modules/installation/pages/general-requirements.adoc (hunk 5 above)
Comment (0rnela) [reply modules/installation/pages/general-requirements.adoc:59]: @keichwa - Sort of. In the last sentence of this very chapter UI is described. From my pow this is not the most elegant place for definition, we should probably start the chapter with it with it, but that would require rewriting of the current first sentence.

**Review by juliogonzalez — APPROVED** [review]
As far as I can tell, the changes look good.

**Review by keichwa — APPROVED** [review]

**Pull request merged**